run-sloth

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
Can you explain in docs what is this script about? what's the purpose and why it exists?

Reply:
Is this  clear enough for you?

Reply:
Now it's better. I know precisely where to look for before analysing the code.